### DISCUSSION TITLE
Show CPU & Battery Temps on screen

### DISCUSSION THREAD

**Discussion by RikshaDriver** [gopro/labs#1098] [Show and tell]
Command to show CPU & Battery Temperatures on-screen... Looks like only HERO13 is supported at the moment.

![GRAB-20240926-154421a](https://github.com/user-attachments/assets/5bf07848-85f2-4f0a-b4cd-88d1ffc590e9)

![image](https://github.com/user-attachments/assets/204a0312-2a8f-486c-9483-666bc3bb71e8)

**Comment by dnewman-gpsw** [comment]
To reduce the overhead on this process use ```$PMSG="CPU:$p:TC BATT:$b:TC"!N!R```  Otherwise is updating way too often the these slow changing fields.

**Reply by RikshaDriver** [reply]
Thanks for that. If I understand correctly, `!R` would update at a rate of 3-5Hz. If we wanted to only update the view every 10 seconds to further reduce load, would we use something like `$PMSG="CPU:$p:TC BATT:$b:TC"!N!10R`

That command doesn't seem to work properly.

**Comment by dnewman-gpsw** [comment]
It is complicated as you have EXPS active.  Any sleep  10 seconds or longer shutdown the camera.  Also !9R adds a pause message, where !2N is silent.  Just use !N!R to keep EXPS running.  With variables and math you can update temperatures separately to EXPS.
